decentraland/unity-renderer
# Feat: added scrollbars in passport navigation tab

## What does this PR change?

Fix #3327 
Adds wearables and emotes in new passport navigation tab

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/passport-collectibles&ENABLE_new_avatar_flow=
2. Open the passport of regular users
3. Go in the second tab
4. Verify that wearables and emotes are correctly displayed

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
